.github/workflows/build.yml

```diff
@@ -13,38 +13,49 @@ jobs:
     name: Build and Test
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: AdoptOpenJDK/install-jdk@v1
```

Comment:
What was the issue with adopt open jdk?

Reply:
Nothing. I just had to change the action.